compiler/ballerina-lang/src/main/java/io/ballerina/projects/DiagnosticResult.java

```diff
@@ -103,4 +108,13 @@ public boolean hasWarnings() {
         hints = Diagnostics.filterHints(diagnostics);
         return hints;
     }
+
+    private Collection<Diagnostic> getInternalExcluded(Collection<Diagnostic> diagnostics) {
+        if (internalExcluded != null) {
+            return hints;
```

Comment:
Why return `hints` here? 🤔

Reply:
:fearful: Oops! My bad. Fixed. Should be `internalExcluded`.